src/lib/sanitize-html.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function (string) {
+  if (typeof string !== 'string') {
+    return '';
+  }
+
+  return string
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;');
+    // https://stackoverflow.com/a/1354491/2601552
```

Comment:
Any reason for not removing this ?

Reply:
Whoops, I just left this in. Will definitely remove it.